### DISCUSSION TITLE
{TITLE}

### DISCUSSION THREAD

**Discussion by DragonSurvival** [Zrips/CMI#1334] [Enhancement]
**Description of issue or feature request:**

Please add {TITLE} for chat and tablist

https://www.spigotmc.org/resources/randompackage-custom-bosses-enchants-170-envoy-tnt-and-more-supports-1-8-8-1-13-2.38501/
I use {TITLES} for this plugin
{TITLES} works on essentials but CMI does not work

#####################################
#####################################
```

https://hastebin.com/wobepemoyo.coffeescript

**Cmi Version (using`/cmi version`):** 8.3.8.1

**Server Type (Spigot/Paperspigot/etc):** spigot

**Server Version (using `/ver`):** 1.12.2

**Relevant plugins (Delete if this isn't needed):** https://www.spigotmc.org/resources/randompackage-custom-bosses-enchants-170-envoy-tnt-and-more-supports-1-8-8-1-13-2.38501/

**Comment by TheWranger** [comment]
What should {TITLE} display?

**Comment by DragonSurvival** [comment]
It is quite similar to Suffix
